code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -2220,17 +2220,15 @@
 	slowdown = GUN_SLOWDOWN_RIFLE_LIGHT_AUTO
 	force = GUN_MELEE_FORCE_RIFLE_HEAVY
 	draw_time = GUN_DRAW_LONG
-	fire_delay = GUN_FIRE_DELAY_NORMAL
+	fire_delay = 1 //delay_slow and delay_normal aren't good here. 1.5 was too slow, 0.6 was too fast. sorry. equivalent is like, normal_delay * 1.66 repeating
```

Comment:
This is equivalent to GUN_FIRE_DELAY_FASTEST

Reply:
it is not. gun_fire_delay_fastest is 0.1. This is **1** flat.

Reply:
SHIT HOLD ON